src/QsCompiler/Core/Transformations/SyntaxTreeTransformation.fs

```diff
@@ -73,6 +58,23 @@ type SyntaxTreeTransformation<'T> private (state: 'T, options: TransformationOpt
 
     new(state: 'T) = new SyntaxTreeTransformation<'T>(state, TransformationOptions.Default)
 
+    // These overrides are here since it is impractical to have them live in the definition of CommonTransformationItems.
+
+    override this.OnArgumentTuple argTuple =
+        this.Namespaces.OnArgumentTuple argTuple
+
+    // These overrides are only here to preserve the functionality of the now deprecated methods.
```

Comment:
Would this be easier to track with a ToDo here?

Reply:
I see you've answered this [here](https://github.com/microsoft/qsharp-compiler/pull/1332#discussion_r799887933).